Add automatic hosts deletion in Insights Inventory (#2502)
theforeman/foreman-documentation: guides/common/modules/proc_using-red-hat-insights.adoc

If you have not set up the API token, you are prompted to create one before using this page.

.Removing hosts from Inventory of Red{nbsp}Hat Insights when they are removed from {Project}
When hosts are removed from {Project}, they can also be removed from the inventory of Red{nbsp}Hat Insights, either automatically or manually.
Use the following procedure to configure it automatically.

.Prerequisite

> * You must have administrator privilege for the organization to enable this setting.

Reviewer: Since the setting is Satellite - wide, you have to be Satellite administrator, not org admin.

Author: Added this one -https://github.com/theforeman/foreman-documentation/pull/2502#discussion_r1366857395

.Procedure
. In the {ProjectWebUI}, navigate to *Configure* > *Inventory Upload*.
. Enable the *Automatic Mismatch Deletion* setting.

The hosts are automatically removed from the Insights Inventory during the next synchronization with {Project} that occurs daily by default.

> If the setting is disabled, you can still delete the hosts from the Inventory manually.

Reviewer: >  you can still delete the hosts from the Inventory manually

Maybe something a bit more specific: you can delete the hosts from the Inventory from `access.redhat.com`?

Author: Addressed :+1:

If you have not set up the API token, you are prompted to create one before using this page.

.Removing hosts from Inventory of Red{nbsp}Hat Insights when they are removed from {Project}
When hosts are removed from {Project}, they can also be removed from the inventory of Red{nbsp}Hat Insights, either automatically or manually.

> To configure it automatically:

Reviewer 2: ```suggestion
To configure it automatically, perform the following steps:
```

. In the {ProjectWebUI}, navigate to *Configure* > *Inventory Upload*.
. Enable the *Automatic Mismatch Deletion* setting.

[NOTE]
====

> You must have administrator privilege for the organization to enable this setting.

Reviewer 3: We should probably list the required permissions in particular here.

Author: Agree and Addressed :+1:

Author: Adding prerequisite and procedure will mix with the heading. It is creating confusion.

Reviewer 3: And use the following text in the Note:
`Your user account has a role assigned that grants the `view_settings` and `edit_settings` permissions.`

Author: Added :+1:

====

> The hosts are automatically removed from the Insights Inventory during the next synchronization with {Project} that occurs in a day by default.

Reviewer 2: ```suggestion
The hosts are automatically removed from the Insights Inventory during the next synchronization with {Project} that occurs daily by default.
```
